.github/workflows/documentation.yml

```diff
@@ -45,20 +54,30 @@ jobs:
         key: ${{ runner.os }}-poetry-3.6-non-full-${{ hashFiles('**/poetry.lock') }}
         restore-keys: ${{ runner.os }}-poetry-3.6-non-full
 
+    - name: Load Yarn Cached Packages ⬇
+      uses: actions/cache@v1
+      with:
+        path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+        key: ${{ runner.os }}-yarn-12.x-${{ hashFiles('newdocs/yarn.lock') }}
+        restore-keys: ${{ runner.os }}-yarn-12.x
+
     - name: Install Dependencies 📦
-      run: make install
-
-    # FIXME: https://github.com/RasaHQ/rasa/issues/6166
-    # - name: Build & Publish Docs 🏃‍♀️
-    #   env:
-    #     GITHUB_DOCS_KEY: ${{ secrets.GITHUB_DOCS_KEY }}
-    #   run: |
-    #     eval "$(ssh-agent -s)"; touch docs_key; chmod 0600 docs_key
-    #     echo "$GITHUB_DOCS_KEY" > "docs_key"
-    #     ssh-add docs_key
-
-    #     git config --global user.email "builds@github-ci.com"
-    #     git config --global user.name "GitHub CI"
-    #     git remote set-url --push origin "git@github.com:${{github.repository}}"
-
-    #     poetry run sphinx-versioning push docs docs . -- -b dirhtml -A html_theme=rasabaster
+      run: make install install-docs
+
+    - name: Build Docs variables 🧶
+      run: cd $DOCS_FOLDER && yarn run variables
```

Comment:
should we do this in the ci workflow as well?

Reply:
yep, good catch